bluesky/run_engine.py

```diff
@@ -107,13 +106,20 @@ class RunEngine:
     loop : asyncio event loop
         e.g., ``asyncio.get_event_loop()`` or ``asyncio.new_event_loop()``
 
-    preprocessors : list
+    preprocessors : list, optional
         Generator functions that take in a plan (generator instance) and
         modify its messages on the way out. Suitable examples include
         the functions in the module ``bluesky.plans`` with names ending in
         'wrapper'.  Functions are composed in order: the preprocessors
         ``[f, g]`` are applied like ``f(g(plan))``.
 
+    context_managers : list, optional
+        Context managers that will be entered when we run a plan. The context
+        managers will be composed in order, much like the preprocessors. If
+        this argument is omitted, we will use a user-oriented handler for
+        SIGINT. The elements of this list will be passed this RunEngine
+        instance as their only argument.
+
```

Comment:
Might be worth explicitly saying that passing an empty list will have no handlers.